packages/rpc/src/iotex/Query.ts

```diff
@@ -0,0 +1,15 @@
+export class Query {
+    rpcUrl: string;
+
+    constructor(rpcUrl: string) {
+        this.rpcUrl = rpcUrl;
+    }
+
+    getAccount = (account: string) => this.uri(`v1/accounts/${account}`);
+    getTransaction = (hash: string) => this.uri(`v1/actions/hash/${hash}`);
+    broadcastTransaction = (data: string) => this.uri('v1/actionbytes/' + data);
```

Comment:
Just a little code style fix: lets use modern string concatenation instead
```broadcastTransaction = (data: string) => this.uri(`v1/actionbytes/${data}`);```

Reply:
Good catch

Reply:
Hello, thanks for the update, issue has been fixed.